chromium_src/chrome/browser/ui/views/bubble/bubble_contents_wrapper.cc

```diff
@@ -0,0 +1,41 @@
+// Copyright (c) 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this file,
+// you can obtain one at http://mozilla.org/MPL/2.0/.
+
+#include <utility>
+
+#include "chrome/browser/ui/views/bubble/bubble_contents_wrapper.h"
+#include "components/sessions/content/session_tab_helper.h"
+#include "content/public/browser/web_contents.h"
+#include "content/public/browser/web_contents_delegate.h"
+#include "content/public/browser/web_contents_observer.h"
+
+#define RenderViewHostChanged                                                \
```

Comment:
I think we need to add a comment for each override we're doing as to why it's needed, so that developers can easily understand the intention of the code when making changes.

Reply:
done